packages/ap-frontend/alliance_platform/frontend/templatetags/form.py

```diff
@@ -72,7 +72,11 @@ def render(self, context: template.Context):
             )
             if help_text:
                 # Help text can be HTML and django docs make it clear this value is not HTML-escaped.
-                help_text = convert_html_string(help_text, self.origin)[0]
+                try:
+                    help_text = convert_html_string(help_text, self.origin)[0]
+                except IndexError:
```

Comment:
Should raising an error in `convert_html_string` - which is then caught here - be the expected behaviour?

Reply:
It's not really easy to do that; the HTML parser basically ignores invalid stuff. This really only happens in cases like:

```
help_text="</script>"
```

this would result in `[]` being returned from `convert_html_string`. If it was

```
help_text="</script>More"
```

it would return `["More"]` and all would be well.

Reply:
Maybe just include a comment to the effect of "`convert_html_string` will silently ignore invalid HTML, which may result in an empty list being returned from non-empty input" either here or on the actual function, just to make that behaviour more explicit?

Reply:
yep good idea - done (rewrote commit & force pushed)